ci/release_notes.md

```diff
@@ -0,0 +1,14 @@
+# Fixes
+- fix a
```

Comment:
Please remove the fix-lines, upgrades and thank yourself :)

Reply:
I always find it weird to thank myself. GitHub shows the contributors anyways so maybe a line like `Huge thanks to all contributors!` would be more appropriate? I will add myself this time but maybe we can change this in the future.